README.md

```diff
@@ -98,27 +102,31 @@ We don't advise you to use the same EFS instance for persisting the data from mu
 
 ### 2. Add the Alfresco Helm repository to helm
 
-```helm repo add alfresco-incubator https://kubernetes-charts.alfresco.com/incubator```
+```
+helm repo add alfresco-incubator https://kubernetes-charts.alfresco.com/incubator
+helm repo add alfresco-stable https://kubernetes-charts.alfresco.com/stable
+```
+
+### 3. Configure domain in your values file
 
-### 3. Define a variable for your Route53 entry that you will use for the deployment
+Pull the values file from the current repo:
 
 ```bash
-export ELB_CNAME="YourDesiredCname.YourRoute53DnsZone"
-#example export ELB_CNAME="alfresco.example.com"
+curl -O https://raw.githubusercontent.com/Alfresco/alfresco-dbp-deployment/master/charts/incubator/alfresco-dbp/values.yaml
 ```
 
+Open it in your favorite text editor and replace all occurences of REPLACEME with your domain.
+
+***Note!***
+Please create a wildcard domain for external-dns (eg. “*.DOMAIN”) as explained [here](https://github.com/Alfresco/alfresco-infrastructure-deployment#nginx-ingress-custom-configuration)
```

Comment:
Explained where? There's no mention of wildcard domains anywhere under "Nginx-ingress Custom Configuration" (or anywhere else in that document for that matter).

Reply:
Yes, this part does not make to much sense. corecting.